plugin.go

```diff
@@ -21,7 +23,15 @@ func (p Plugin) buildCommand() *exec.Cmd {
 	// Set the default of distributions in here
 	// as CLI package still has issues with string slice defaults
 	distributions := []string{"sdist"}
-	if len(p.Distributions) > 0 {
+	// Sanitize bad dist values
+	inputDists := make([]string, 0, len(p.Distributions))
```

Comment:
This is really unrelated to the issue and beside that I'm not sure if I would like to manipulate that list like that.